WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionSource.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui
+
+enum class JetpackConnectionSource(private val value: String) {
+    NOTIFICATIONS("notifications"), STATS("stats");
+
+    override fun toString(): String {
```

Comment:
Suggestion (💡): This `toString()` function can be further simplified to: `override fun toString() = value`

Reply:
See 11837b4

Reply:
👋 @zwarm ! This is probably the wrong commit, right?